modules/reference/pages/rpk/rpk-connect/rpk-connect-studio-sync-schema.adoc

```diff
@@ -0,0 +1,27 @@
+= rpk connect studio sync-schema
+
+Synchronizes the schema of this Redpanda Connect instance with a studio session.
+
+This sync allows custom plugins and templates to be configured and linted correctly within Benthos studio.
```

Comment:
```suggestion
This sync allows custom plugins and templates to be configured and linted correctly within Benthos Studio.
```
Not sure if this should be just Studio. Guessing eventually this changes to Redpanda Console?

Reply:
this is still tbd. But the change to Benthos Studio looks good